### ISSUE TITLE
Unable to load epub where paths to resources are not normalized

### ISSUE THREAD

**Issue by kamilgolunski** [logiclord/epubReader#23]
Loading epub fails when path to resources (like images or fonts) are not normalized, e.g "img/../img/cover.jpg" or "css/../fonts/font.ttf". 

I get error on line:
https://github.com/logiclord/epubReader/blob/master/js/epubReaderFileFacilitator.js#L69, 
data is called on 'null', I think that "imgpath" should be normalized.

**Comment by logiclord** [comment]
Good catch. Yes, this seems like an issue. Here I have assumed that JSZip Library will take care of such path.
